com.steamgriddb.SGDBoop.yml

```diff
@@ -0,0 +1,25 @@
+app-id: com.steamgriddb.SGDBoop
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: SGDBoop
+finish-args:
+  - --share=network
+  - --filesystem=~/.steam/steam/config/loginusers.vdf:ro # Used for reading SteamID
+  - --filesystem=~/.steam/steam/userdata:rw # For saving custom images except icons
+  - --filesystem=~/.steam/steam/appcache/librarycache:rw # For saving icons
```

Comment:
I assume this is not going to work with flatpaked Steam though? It should also support `~/.var/app/com.valvesoftware.Steam`.

Reply:
I've tested with the flatpaked Steam seems to initialize itself in `~/.steam/` just like the other versions.

Reply:
Flatpaked Steam makes its data visible at `~/.steam` only within its own sandbox with a bind mount, it's not using that directory otherwise.

Reply:
Just tested with a fresh install with `flatpak --user install flathub com.valvesoftware.Steam` and the `~/.steam` directory is created outside the sandbox (and is being read from/written to by flatpak Steam). Maybe it has something to do with the wrapper? Or perhaps I'm misunderstanding something.

Reply:
Have you changed some global permissions? I launched Steam right now and there's no `~/.steam` outside Steam sandbox, i.e. it's only visible to Steam:

```
[bpiotrowski@risuku ~]$ stat .steam
stat: cannot statx '.steam': No such file or directory
[bpiotrowski@risuku ~]$ flatpak run --command=ls com.valvesoftware.Steam ~/.steam
bin  bin32  bin64  registry.vdf  root  sdk32  sdk64  starting  steam  steam.pid  steam.pipe  steam.token
```

Reply:
Yeah, looks like i messed up. I'll update the program to detect flatpaked Steam.